src/react-apps/applications/ux-editor/src/containers/DroppableDraggableContainer.tsx

```diff
@@ -144,7 +145,7 @@ class DroppableDraggableContainer extends React.Component<IDroppableDraggableCon
           border: '1px solid #ccc',
           padding: '1em',
           marginBottom: -1,
-          backgroundColor: isOver ? 'lightgrey' : 'white',
+          backgroundColor: isOver ? 'white' :  altinnTheme.altinnPalette.primary.greyLight,
```

Comment:
Is it right that these are switched?

Reply:
Yes, the background of the ui-editor should be greyLight and therefor the DroppableDraggableContainer can't be the same color on drag to mark where the object should be dropped.